Frameworks/CoreText/CTFramesetter.mm

```diff
@@ -48,8 +48,10 @@ CTFrameRef CTFramesetterCreateFrame(CTFramesetterRef framesetterRef, CFRange ran
         range.length = [typesetter->_string length] - range.location;
     }
 
-    StrongId<_CTFrame> ret = _DWriteGetFrame(static_cast<CFAttributedStringRef>(typesetter->_attributedString.get()), range, frameRect);
-    ret->_path.reset(CGPathRetain(path));
+    woc::StrongCF<__CTFrame*> ret;
+    ret =
```

Comment:
This is due to the minor shortfall of StrongCF, assignment of a  return value will cause it to invoke a +0 recount and since we don't own the object, it will end up with over releases.